mmdet/datasets/pipelines/transforms.py

```diff
@@ -2813,6 +2814,42 @@ def get_indexes(self, dataset):
         """
         return random.randint(0, len(dataset))
 
+    def gen_masks_from_bboxes(self, bboxes, img_shape):
```

Comment:
Does the dev-3.x branch need to be updated in sync?

Reply:
Yes, we should migrate the change.